test/qa/test_connect.py

```diff
@@ -483,52 +483,57 @@ def test_status_disconnected(tech, proto, obfuscated):
 
 @pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
 @pytest.mark.flaky(reruns=2, reruns_delay=90)
-@timeout_decorator.timeout(70)
+@timeout_decorator.timeout(60)
 def test_status_connected(tech, proto, obfuscated):
     lib.set_technology_and_protocol(tech, proto, obfuscated)
 
     with lib.Defer(sh.nordvpn.disconnect):
         name, hostname = server.get_hostname_by(technology=tech, protocol=proto, obfuscated=obfuscated)
         sh.nordvpn.connect(hostname.split(".")[0])
+
         connect_time = int(time.time())
 
-        assert hostname in sh.nordvpn.status()
+        data_10MB = requests.get("http://speedtest.tele2.net/10MB.zip")
+        destination_file = "file_10MB"
+        if data_10MB.status_code == 200:
```

Comment:
need to handle failure of downloading above file, maybe `assert` instead of `if` would be appropriate here